app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -176,6 +176,7 @@ class LuciBuildService {
     }
     properties ??= <String, dynamic>{};
     properties.addAll(<String, String>{
+      'git_branch': pullRequest.base!.ref!.replaceAll('refs/heads/', ''),
```

Comment:
Do we have a property to identify this is presubmit?

Reply:
No, we don't. You could infer that if `git_ref.contains('refs/pull')` it's presubmit, but I wouldn't recommend it.

In the recipe, you can check the buildbucket bucket == try, which is what the devicelab recipe does.